core/urls.py

```diff
@@ -60,6 +60,7 @@ class PaginatedSitemap(GenericSitemap):
     path("dsfr/", include(("dsfr_hacks.urls", "dsfr_hacks"), namespace="dsfr_hacks")),
     path("", include(("qfdmo.urls", "qfdmo"), namespace="qfdmo")),
     path("", include(("qfdmd.urls", "qfdmd"), namespace="qfdmd")),
+    path("", include(("data.urls", "data"), namespace="data")),
```

Comment:
Est-ce qu'on ne commencerait pas à namespacer nos URLs ? 
```suggestion
    path("/data", include(("data.urls", "data"), namespace="data")),
```

Reply:
yes mais pas dans cette PR si ça te convient

Reply:
aucun soucis